packages/cozy-stack-client/src/OAuthClient.js

```diff
@@ -111,6 +112,18 @@ export default class OAuthClient extends CozyStackClient {
   async register() {
     if (this.isRegistered()) throw new Error('Client already registered')
 
+    const mandatoryFields = ['redirectURI']
+    const fields = Object.keys(this.oauthOptions)
+    const missingMandatoryFields = difference(mandatoryFields, fields).join(
+      ', '
+    )
```

Comment:
Alternative name and method :

```
const missingFields = mandatoryFields.filter(fieldName => this.oauthOptions[fieldName])
```

I would join in the error since it's joined only for formatting.